src/components/MyDatePicker/MyDatePicker.jsx

```diff
@@ -0,0 +1,88 @@
+import { useState } from "react";
+import Flatpickr from "react-flatpickr";
+import "flatpickr/dist/flatpickr.min.css";
+import dateIcon from "@/assets/icons/contractsIcons/dateIcon.svg";
+import styles from "./styles";
+
+const Armenian = {
```

Comment:
create consts folder and for example myDatePicker.js, and keep such arrays there